server/test/utils.ts

```diff
@@ -60,11 +50,19 @@ async function createUser({
     github_username: github_username ?? randomEmptyChance(20, randUserName()),
     website: website ?? randomEmptyChance(20, randUrl()),
   };
+}
 
+async function createUser(options: Partial<UserObject> = {}) {
+  const userObject = makeUserObject(options);
   // insert into DB after creation
-  const user = await User.create(userObject);
+  return await User.create(userObject);
+}
 
-  return user;
+async function createManyUsers(numberOfUsers: number): Promise<User[]> {
+  const userArray = Array.from({ length: numberOfUsers }, makeUserObject);
+  return await Db.sequelize.transaction(async transaction => {
+    return await User.bulkCreate(userArray, { transaction });
```

Comment:
unnecessary return here - can just be `await User.bulkCreate ...`

Reply:
I may be mistaken, but I do believe both `return`s are necessary. The opposite could be said for both `await`s though.

Reply:
ah yup, you're right - misunderstood how these were getting returned.  thanks for clearing that up!